src/utils/notification.ts

```diff
@@ -8,21 +9,39 @@ export type DefaultRequestError = {
   status?: number;
 };
 
-export function displayDefaultNotificationFlow<T = unknown>(res?: RTKResponse<T>, success?: () => void) {
-  if (res && 'error' in res && 'data' in res.error) {
-    if (res.error.data) {
+export function defaultNotificationFlow<T = unknown>(
```

Comment:
* Looks like it will have a lot of different responsibilities, instead of simply choosing flow. Probably better to extract `getErrorFromResponse` or something like that instead
* Worth to note, that this function is used by Cloud version too, so I would probably avoid changing the name
* Given that this function is synchronous and can't be async - does it make sense to have the callbacks?

Reply:
You mean create another function `getErrorFromResponse` and use it in such cases?

Reply:
Refactored error handing. Looks better now, I think. Do you like it? Or should I move `success` callback to `then` as well?

Reply:
I think that it would be better to move it to `then` as well, to keep single mechanism :)

Reply:
Got it, so we might want to change cloud as well